### DISCUSSION TITLE
Is the server down?

### DISCUSSION THREAD

**Discussion by kamen90** [rustdesk/rustdesk#3934] [General]
Hi, I am not able to connect to any of the remote devices today. I see other users have the same issues: https://www.reddit.com/r/rustdesk/comments/12ca42x/just_in_server_is_down/

Can you please share some insights on the issue? Thank you!

**Comment by vladi-bsgr** [comment]
I hope it will be online soon.

**Reply by itestings** [reply]
I Also have issue with 1.1.9 ... but made test upgrade to night 1.2.0 and i'm able to connect ... Think that is a certificate issue or something like that on 1.1.9
